File: pages/cgi-bin/env.py
#!/usr/bin/python3
import os
print ("Content-Type: text/html; charset=UTF-8\r\n\r\n")
print("<!DOCTYPE html>")
print("<html lang=\"en\">")
print("<head>")
print("    <meta charset=\"UTF-8\">")
print("    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">")
print("    <link rel=\"shortcut icon\" href=\"favicon.ico\" type=\"image/x-icon\">")
print("</head>")
print("<body>")
print("<ul>")
for name, value in os.environ.items():
    print("<li>")
    print("{0}: {1}".format(name, value))
    print("</li>")
print("</ul>")
print("</body>")
print("</html>")

File: pages/cgi-bin/env.php
#!/usr/bin/php-cgi

<!DOCTYPE html>
<html lang='en'>
<head>
    <meta charset='UTF-8'>
    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
    <title>Message Received</title>
    <style>
        h1, p {  width: 50%; margin: 20px auto;}
    </style>
</head>
<body>
<?php
    echo "<pre>";
    foreach ($_SERVER as $key => $value) {
        echo "$key: $value\n";
    }
    echo "</pre>";
?>
</body>
</html>
